lib/solidus_paypal_braintree/engine.rb

```diff
@@ -23,32 +21,22 @@ def self.activate
 
     config.to_prepare(&method(:activate).to_proc)
 
-    if SolidusSupport.frontend_available?
-      config.assets.precompile += [
-        'solidus_paypal_braintree/checkout.js',
-        'solidus_paypal_braintree/frontend.js',
-        'spree/frontend/paypal_button.js',
-        'spree/frontend/apple_pay_button.js'
-      ]
-      paths["app/controllers"] << "lib/controllers/frontend"
-      paths["app/views"] << "lib/views/frontend"
-    end
+    config.assets.precompile += [
```

Comment:
We should still ask if the `spree_frontend` and `spree_backend` gems are actually available, by using the old way:

```rb
def self.frontend_available?
  defined?(Spree::Frontend::Engine) == "constant"
end

def self.backend_available?
  defined?(Spree::Backend::Engine) == "constant"
end
```